src/apis/clubs/services/clubs.service.ts

```diff
@@ -96,4 +186,63 @@ export class ClubsService {
 
     return [clubs, count];
   }
+
+  async findOneOrNotFound(clubId: number): Promise<ClubDto> {
+    const existClub = await this.clubRepository.findOneBy({
+      id: clubId,
+      status: ClubStatus.Active,
+    });
+
+    if (!existClub) {
+      throw new HttpNotFoundException({
+        code: COMMON_ERROR_CODE.RESOURCE_NOT_FOUND,
+      });
+    }
+
+    return new ClubDto(existClub);
+  }
+
+  async bulkCreateClubTagLinks(
+    createClubTagLinkDtos: CreateClubTagLinkDto[],
+  ): Promise<ClubTagLink[] | undefined> {
+    if (!createClubTagLinkDtos.length) {
+      return;
+    }
```

Comment:
```suggestion
  ): Promise<ClubTagLink[]> {
    if (!createClubTagLinkDtos.length) {
      return [];
    }
```
동일하게 배열을 리턴하는건 어떤가요?

Reply:
일관성 있는 처리가 되겠네요 수정하겠습니다 항상 꼼꼼히 봐주셔서 감사합니다!!🙇